integration/oasis/handler.go

```diff
@@ -45,6 +48,15 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// check whether orig and dest is reachable
+	if len(route.Routes) == 0 {
+		glog.Info("Orig and destination is not reachable for request.")
```

Comment:
Nit: Better to provide more information here, i.e. provide both origin and destination coordinates. The idea is from https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures but I think it's also applicabe for any logs.